armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -669,7 +667,9 @@ def _checkMass(self, origDetailedNDens, origVolume, newDetailedNDens, newVolume,
                 )
             else:
                 # should not conserve mass here as it is structural material above active fuel
-                self.assertAlmostEqual(newMass / prevMass, 0.99, msg=f"{c}, {c.parent}")
+                self.assertAlmostEqual(
+                    newMass / prevMass, 0.99999999, msg=f"{c}, {c.parent}"
```

Comment:
Curious why the precision increase here? I don't think it hurts anything, but am just curious.

Reply:
Reverted